appinventor/appengine/tests/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectServiceTest.java

```diff
@@ -0,0 +1,19 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
```

Comment:
No need to keep the Google copyright since this file was written at MIT. Update the years accordingly.